### DISCUSSION TITLE
Why can't I add remote url

### DISCUSSION THREAD

**Discussion by firekula** [gitkraken/vscode-gitlens#2546] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/44957428/223361737-9cb0254b-4791-4f94-9b41-f88176b38681.png)

**Comment by eamodio** [comment, marked as answer]
FYI, this was a bug and has been fixed -- you can track it here: https://github.com/gitkraken/vscode-gitlens/issues/2553

Thanks!
